sdk/src/wallet/bindings/nodejs/lib/Account.ts

```diff
@@ -394,6 +395,47 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Verifies a message with an ed25519 signature.
+     */
+    async verifyEd25519Signature(
+        signature: Ed25519Signature,
+        message: HexEncodedString,
+    ): Promise<boolean> {
+        const response = await this.messageHandler.callAccountMethod(
+            this.meta.index,
+            {
+                name: 'verifyEd25519Signature',
+                data: {
+                    signature,
+                    message,
+                },
+            },
+        );
+        return JSON.parse(response).payload;
+    }
+
+    /**
+     * Verifies a message with an Secp256k1Ecdsa public key and signature.
```

Comment:
Also here, the signature gets verified, not the message

Reply:
Done